.circleci/config.yml

```diff
@@ -98,6 +98,12 @@ workflows:
             - js_build
             - build_server_pdfs
             - build_api_docs
+      - optimize-images:
+          requires:
+            - build              
+          # filters: 
+          #   branches:
+          #     only: master 
```

Comment:
currently testing out the new job, will comment out afterwards to have `optimize-images` run only when merging with master

Reply:
uncommented filter for `minimize-images` job to have it run only when we merge into master due to successful testing of `minimize-images` in preview branch